### DISCUSSION TITLE
Switching Download Folder

### DISCUSSION THREAD

**Discussion by Tvastr-ops** [z-huang/music#590] [Q&A] [answered]
Is it Possible to download songs in specific folders ?

**Comment by z-huang** [comment, marked as answer]
No, but you can access downloaded files. See readme.

**Reply by selurvedu** [reply]
What about 0.5.0?

**Comment by z-huang** [comment]
Since 0.5.0, you can no longer access downloaded files. However, the download experience has been improved.
